src/main/java/teammates/common/datatransfer/FeedbackQuestionRecipientAttributes.java

```diff
@@ -0,0 +1,89 @@
+package teammates.common.datatransfer;
+
+import java.util.Objects;
+
+/**
+ * The data transfer class for Recipient.
```

Comment:
"The data transfer class for ..." has little meaning when there is no persistable entity involved